### ISSUE TITLE
How configure hosts allowed to relay

### ISSUE THREAD

**Issue by LordOfTheRams** [mailcow/mailcow-dockerized#1790]
I'm running a fresh install. Mailcow runs on an internal network IP (192.168.1.4) behind NAT. When I send mail from another IP (192.168.1.5) on this network I would have expected it to fail, because I did not configure anything. The log says that 192.168.1.5 ist whitelisted, but in the UI everything is just empty. I checked postfix main.cf file which only allows localhost to relay. So where is the configuration that allows the server to relay mail from the internal network? 

Log messages:
```
WHITELISTED [192.168.1.5]:51366
CONNECT from [192.168.1.5]:51366 to [172.22.1.4]:25
```

**Comment by LordOfTheRams** [comment]
Found part of the solution myself. The right config file for postfix is in /opt/postfix. There I found:

`mynetworks = 127.0.0.0/8 [::ffff:127.0.0.0]/104 [::1]/128 10.0.0.0/8 172.16.0.0/12 192.168.0.0/16 [fe80::]/10 [fc00::]/7`

This is where relaying is allowed for the local network. Still the remaining questions

- is this line managed from the UI? I guess not. Which is ok for me, it is just easy to get confused with the forwarding hosts and the relay hosts.
- main.cf also exists in the data/conf/postfix directory of the host system. is this the right place to edit mynetworks and then restart postfix?
